src/main/java/com/glodblock/github/crossmod/waila/part/SpeedWailaDataProvider.java

```diff
@@ -0,0 +1,48 @@
+package com.glodblock.github.crossmod.waila.part;
+
+import appeng.api.parts.IPart;
+import appeng.integration.modules.waila.part.BasePartWailaDataProvider;
+import appeng.util.Utility;
+import com.glodblock.github.common.parts.PartSharedFluidBus;
+import com.glodblock.github.util.NameConst;
+import mcp.mobius.waila.api.IWailaConfigHandler;
+import mcp.mobius.waila.api.IWailaDataAccessor;
+import net.minecraft.client.resources.I18n;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
+
+import java.util.List;
+
+public class SpeedWailaDataProvider extends BasePartWailaDataProvider {
+
+    @Override
+    public List<String> getWailaBody(
+        final IPart part,
+        final List<String> currentToolTip,
+        final IWailaDataAccessor accessor,
+        final IWailaConfigHandler config) {
+        if (part instanceof PartSharedFluidBus) {
+            part.readFromNBT(accessor.getNBTData());
+            currentToolTip.add(String.format("%s: %s mB", I18n.format(NameConst.WAILA_SPEED), Utility.formatNumbers(((PartSharedFluidBus) part).calculateAmountToSend())));
```

Comment:
mB/t or mB/s ?

Reply:
![image](https://user-images.githubusercontent.com/33802162/204505733-54379022-c379-46e5-9e70-50064c61fc46.png)

Reply:
use the 5 ticks as display value